client/src/app/core/core-services/active-meeting-id.service.ts

```diff
@@ -50,7 +50,26 @@ export class ActiveMeetingIdService {
         this.lifecycleService.openslidesShutdowned.subscribe(() => this.shutDown());
     }
 
-    public start(): void {
+    private start(): void {
+        this.initialize();
+        this.initSubscription();
+    }
+
+    private shutDown(): void {
+        this.currentMeetingId = null;
+        this.meetingIdSubject.next(undefined);
```

Comment:
null instead of undefined - this is consistent. And `null` indicates, that the service actively knows, that there is no meeting. Undefined would be like "not yet initialized".